.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Python 3 on Linux
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    strategy:
+      max-parallel: 4
```

Comment:
why max parallel?

Reply:
Artifact from when I had tried to run across 3 OS's. Can be removed, though it has no effect here as the matrix only expands to 4 jobs.